sky/clouds/service_catalog/data_fetchers/fetch_lambda_cloud.py

```diff
@@ -16,7 +16,8 @@
 ENDPOINT = 'https://cloud.lambdalabs.com/api/v1/instance-types'
 DEFAULT_LAMBDA_KEYS_PATH = os.path.expanduser('~/.lambda_cloud/lambda_keys')
 
-# This is the list that Lambda Labs gave us.
+# List of all possible regions. The best way to obtain this list is to ask
+# Lambda. Could not find this list online.
```

Comment:
Is this true?

<img width="600" alt="Screen Shot 2023-06-02 at 09 46 05" src="https://github.com/skypilot-org/skypilot/assets/592670/e6bd2390-96ca-4143-a984-76c65f361020">

In fact, the above shows `us-midwest-1`, but it's not included here. Can we check the screenshot (needs scroll down) and use that list?

Reply:
Oh, I didn't know about this! This is very nice. Let me fix.

I guess this is what happens when you don't ever use the Lambda UI to launch instances :))

Reply:
Ok, I checked. Only `us-midwest-1` is missing.

Reply:
Fixed! I tried testing the catalog with us-midwest-1 added, but there were no vms available in that region.